content/environment_variables.tex

```diff
@@ -30,14 +30,32 @@
     \tabularnewline\hline
 %%
 \EnvVarDecl{SHMEM\_SYMMETRIC\_SIZE}
-    & Non-negative integer
-    & Number of bytes to allocate for symmetric heap
+    & Positive decimal with a character suffix or a positive integer with
+    optional character suffix
+    & Controls the size of the symmetric heap per \ac{PE}. The value
+      set in this environment variable is interpreted as number
+      of bytes, unless the number is followed by a single
+      character that acts as a multiplier, where:
+      \begin{itemize}
+        \item k or K multiplies by \(2^{10}\)  (kilobytes)
+        \item m or M multiplies by \(2^{20}\)  (megabytes)
+        \item g or G multiplies by \(2^{30}\)  (gigabytes)
+        \item t or T multiplies by \(2^{40}\)  (terabytes)
+      \end{itemize}
+      For example, the string 20m is equivalent to the integer value 20971520,
+      or 20 megabytes. Similarly the string 3.1g is equivalent to the integer
+      value 3250585. Only one multiplier is recognized and a positive decimal
```

Comment:
I think the above number is incorrect -- 3.1g = 3.1 * 2^30 = 3328599654.4

Should we specify what to do with the 4/10 of a byte?